pkg/service/model_build_target_group.go

```diff
@@ -362,7 +362,14 @@ func (t *defaultModelBuildTask) buildTargetGroupBindingSpec(ctx context.Context,
 	if targetType == elbv2api.TargetTypeInstance {
 		targetPort = intstr.FromInt(int(port.NodePort))
 	}
-	tgbNetworking := t.buildTargetGroupBindingNetworking(ctx, targetPort, preserveClientIP, *hc.Port, port.Protocol)
+	defaultSourceRanges := []string{"0.0.0.0/0"}
```

Comment:
nit: since defaultSourceRanges is only used in buildTargetGroupBindingNetworking, 
i'd suggest pass `scheme` into buildTargetGroupBindingNetworking instead, and construct defaultSourceRanges within there. either inline or have a function for build default source ranges.